backend/src/mirrors_qa_backend/routes/tests.py

```diff
@@ -31,11 +36,11 @@ def list_tests(
     sort_by: Annotated[TestSortColumnEnum, Query()] = TestSortColumnEnum.requested_on,
     order: Annotated[SortDirectionEnum, Query()] = SortDirectionEnum.asc,
 ) -> schemas.TestsList:
-    result = db.tests.list_tests(
+    result = tests.list_tests(
         session,
         worker_id=worker_id,
         country=country,
-        status=status,
+        statuses=status,
```

Comment:
Should we name the query param `statuses` as it's a list?

Reply:
I think `status` is better for the user seeing as they would be specifying it as `&status=MISSED&status=PENDING`

Reply:
Yeah that's what I figured as well. I also prefer `status` so let's keep it.